src/test/java/seedu/mtracker/console/AddForexParserTest.java

```diff
@@ -0,0 +1,125 @@
+package seedu.mtracker.console;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class AddForexParserTest {
+
+    public static final int PARAMETER_SIZE = 7;
+    private static final String STR_SEP = "%1$s";
```

Comment:
Rename to SEPARATOR_SPECIFIERS or something %1$s is like a format specifier

Reply:
Done